src/components/molecules/ImageInput/ImageInput.tsx

```diff
@@ -24,94 +24,82 @@ interface ImageInputProps extends ContainerClassName {
+  imageClassName,
+  name,
+  error,
+  disabled,
+  setValue,
+  register,
+}) => {
+  const imageUrl = useMemo(
+    () =>
+      (image && image.length > 0 && URL.createObjectURL(image[0])) ||
       remoteImageUrl,
-      containerClassName,
-      imageClassName,
-      name,
-      error,
-      disabled,
-      setValue,
-      register,
-    },
-    ref
-  ) => {
-    const imageUrl = useMemo(
-      () =>
-        (image && image.length > 0 && URL.createObjectURL(image[0])) ||
-        remoteImageUrl,
-      [image, remoteImageUrl]
-    );
+    [image, remoteImageUrl]
+  );
 
-    const {
-      loading,
-      errorMessage,
-      handleFileInputChange,
-    } = useImageInputCompression(register, error?.message, name);
+  const {
+    loading,
+    errorMessage,
+    handleFileInputChange,
+  } = useImageInputCompression(register, error?.message, name);
 
-    const handleFileInputChangeWrapper = useCallback(
-      async (event: ChangeEvent<HTMLInputElement>) => {
-        const [url, compressedFile] = await handleFileInputChange(event);
-        if (!compressedFile || !url) return;
+  const handleFileInputChangeWrapper = useCallback(
+    async (event: ChangeEvent<HTMLInputElement>) => {
+      const [url, compressedFile] = await handleFileInputChange(event);
+      if (!compressedFile || !url) return;
 
-        setValue(name, [compressedFile], false);
+      setValue(name, [compressedFile], false);
 
-        if (remoteUrlInputName) {
-          setValue(remoteUrlInputName, url, true);
-        }
-      },
-      [handleFileInputChange, name, remoteUrlInputName, setValue]
-    );
+      if (remoteUrlInputName) {
+        setValue(remoteUrlInputName, url, true);
+      }
+    },
+    [handleFileInputChange, name, remoteUrlInputName, setValue]
+  );
 
-    return (
-      <>
-        <div
-          className={classNames(
-            `image-input default-container`,
-            containerClassName,
-            {
-              disabled: loading,
-            }
-          )}
-        >
-          {imageUrl ? (
-            <img
-              className={`default-image ${imageClassName}`}
-              src={imageUrl}
-              alt="upload"
-            />
-          ) : (
-            <div className="centered-flex empty">
-              <h6 className=" text" style={{ fontWeight: "lighter" }}>
-                Click to upload an image
-              </h6>
-            </div>
-          )}
-          <input
-            disabled={disabled || loading}
-            type="file"
-            accept={ACCEPTED_IMAGE_TYPES}
-            onChange={handleFileInputChangeWrapper}
-            className="default-input"
+  return (
+    <>
+      <div
+        className={classNames(
+          `image-input default-container`,
+          containerClassName,
+          {
+            disabled: loading,
+          }
+        )}
+      >
+        {imageUrl ? (
+          <img
+            className={`default-image ${imageClassName}`}
```

Comment:
Same, use outside `classNames()` for composite classes. Makes it easily diffable and maintainable Can't hurt to add `ImageInput__upload-image` or similar class even if not changing the SCSS

Reply:
That is just an indentation change. I didn't write any of that code. 
And it's pretty much all legacy here. I feel a very strong need to remove that just don't want to start that now.

Reply:
>  I didn't write any of that code.

This https://github.com/sparkletown/sparkle/pull/2201#discussion_r700134601 isn't a note on code you may have written, it is more like a `if you would be so kind to also do this since you're changing code in the vicinity"

I see it in the same venue as deleting some obsolete function that some one else wrote but isn't used anymore. Only in this case, it's connected with the code you write.

On the other side, if  you can't touch code some one else wrote, than all your changes are also in a code some one else wrote. So it's more like, do you think it is too far [OOS] and/or will it take too much time to do so?